src/ngx_http_lua_kong_module.c

```diff
@@ -158,4 +158,33 @@ ngx_http_lua_kong_merge_loc_conf(ngx_conf_t *cf, void *parent, void *child)
     return NGX_CONF_OK;
 }
 
+ngx_flag_t
+ngx_http_lua_kong_get_next_upstream_mask(ngx_http_request_t *r,
+    ngx_flag_t upstream_next)
```

Comment:
It seems this function is not used?

Reply:
https://github.com/Kong/kong/pull/14029/files#diff-87978139f4c7701a35f1c2eaf5b8f71ccc16f9549571ab6c97dc127ba6be31bbR10 in here